service-front/module/Application/src/Validators/PassportDateValidator.php

```diff
@@ -26,10 +28,15 @@ private function fiveYearValidity(string $date): bool
     {
         try {
             $now = time();
-            $effectiveExpiry = date('Y-m-d', strtotime('+5 year', strtotime($date)));
+            $expiryDate = strtotime($date);
+            if ($expiryDate === false) {
+                return false;
+            }
+            $effectiveExpiry = date('Y-m-d', strtotime(self::EXPIRY_ALLOWANCE, $expiryDate));
 
             return $now < strtotime($effectiveExpiry);
         } catch (\Exception $exception) {
+            error_log($exception->getMessage());
```

Comment:
If we actually want to capture this, we should use the `LoggerInterface` service, but I suspect this was just here for debugging?